[2025-05-14T05:27:03.384+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T05:20:00+00:00 [queued]>
[2025-05-14T05:27:03.397+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T05:20:00+00:00 [queued]>
[2025-05-14T05:27:03.398+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T05:27:03.398+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-14T05:27:03.399+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T05:27:03.418+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Extract_Data> on 2025-05-14 05:20:00+00:00
[2025-05-14T05:27:03.426+0000] {standard_task_runner.py:52} INFO - Started process 274 to run task
[2025-05-14T05:27:03.429+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Sales_Performance_ETL', 'Extract_Data', 'scheduled__2025-05-14T05:20:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Hisham_Audiputra_Suharto_DAG.py', '--cfg-path', '/tmp/tmph77coumj', '--error-file', '/tmp/tmp2ymyrd2w']
[2025-05-14T05:27:03.432+0000] {standard_task_runner.py:80} INFO - Job 3: Subtask Extract_Data
[2025-05-14T05:27:03.517+0000] {task_command.py:371} INFO - Running <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T05:20:00+00:00 [running]> on host 2b7490cbbc81
[2025-05-14T05:27:03.599+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Sales_Performance_ETL
AIRFLOW_CTX_TASK_ID=Extract_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-14T05:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-14T05:20:00+00:00
[2025-05-14T05:27:03.602+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-14T05:27:03.604+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Extract_Data.py']
[2025-05-14T05:27:03.616+0000] {subprocess.py:85} INFO - Output:
[2025-05-14T05:27:04.391+0000] {subprocess.py:92} INFO - Warning: Looks like you're using an outdated `kagglehub` version, please consider updating (latest version: 0.3.12)
[2025-05-14T05:27:05.263+0000] {subprocess.py:92} INFO - Downloading from https://www.kaggle.com/api/v1/datasets/download/zahidmughal2343/supplement-sales-data?dataset_version_number=1...
[2025-05-14T05:27:05.614+0000] {subprocess.py:92} INFO -   0%|          | 0.00/65.2k [00:00<?, ?B/s]100%|██████████| 65.2k/65.2k [00:00<00:00, 210kB/s]100%|██████████| 65.2k/65.2k [00:00<00:00, 209kB/s]Extracting model files...
[2025-05-14T05:27:05.641+0000] {subprocess.py:92} INFO - 
[2025-05-14T05:27:05.642+0000] {subprocess.py:92} INFO - Path to dataset files: /home/***/.cache/kagglehub/datasets/zahidmughal2343/supplement-sales-data/versions/1
[2025-05-14T05:27:05.644+0000] {subprocess.py:92} INFO - Path to dataset files: /home/***/.cache/kagglehub/datasets/zahidmughal2343/supplement-sales-data/versions/1
[2025-05-14T05:27:05.666+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-14T05:27:05.711+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Sales_Performance_ETL, task_id=Extract_Data, execution_date=20250514T052000, start_date=20250514T052703, end_date=20250514T052705
[2025-05-14T05:27:05.734+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-14T05:27:05.789+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
